### PR TITLE
New HTML Tab (Issue 24)

### PR DESCRIPTION
**Pull request by Nhamakami7**
Created a new HTML tab that displays the HTML content for the current level.

### CODE REVIEW

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
I can review after @Kyle-Pu does :)

File: src/components/levels.js [hunk 1]
```diff
@@ -53,16 +53,16 @@ This CSS will turn all Zebras a deep shade of purple!`,
...
-        new stickers.Acacia('Aiden', 15, 50, "#1d0b08", "plant"),
-        new stickers.Acacia('Alicia', 30, 40, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Edward', 78, 6, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Elizabeth', 50, 12, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Ella', 65, 40, "#1d0b08", "plant"),
-        new stickers.ElephantGrass('Eduardo', 10, 75, "#1d0b08", "plant"),
-        new stickers.Zebra('zaira', 50, 50, "#fffcf5")
+        new stickers.Boabab('Bob', 10, 30, "#1d0b08", "boabab", "plant"),
+        new stickers.Boabab('Bill', 65, 75, "#1d0b08", "boabab", "plant"),
+        new stickers.Acacia('Ace', 10, 5, "#1d0b08", "acadia", "plant"),
```
Comment (Kyle-Pu) [inline src/components/levels.js:58]:
```suggestion
        new stickers.Acacia('Ace', 10, 5, "#1d0b08", "acacia", "plant"),
```

File: src/components/Stickers.js [hunk 2]
```diff
@@ -6,11 +6,12 @@ import { AcaciaSVG } from './assets/Acacia.js';
 import { ElephantGrassSVG } from './assets/ElephantGrass.js';
 
 class sticker{
```
Comment (Kyle-Pu) [inline src/components/Stickers.js:8]: I don't know what convention this project follows (@reginawang3495?), but I usually capitalize class names.

**Review by Kyle-Pu — CHANGES_REQUESTED** [review]
Looks pretty good and works real smoothly. I think it might look slightly better if the tab buttons have less space between them, but that's just personal preference.

File: src/components/Stickers.js (hunk 2 above)
Comment (reginawang3495) [reply src/components/Stickers.js:8]: yeeee same, this repository is a bit legacy, which is why it's like this, but I agree

File: src/components/levels.js (hunk 1 above)
Comment (reginawang3495) [reply src/components/levels.js:58]: we probably don't want to be pass in the component's name to be a class variable. This is duplication of code, and is prone to human error (see case study 1, this one that kyle pointed out :0). I look at this is think that there's probably a way to get an object's classname ;)

lmk if you need more help, but overall, we just shouldn't have the `component_name` variable overall

**Review by Kyle-Pu — APPROVED** [review]
Looks smooth and works nicely.

**Pull request merged**